src/main/resources/rulesets/checkstyle/rules.xml

```diff
@@ -52,6 +52,15 @@
   <module name="org.openhab.tools.analysis.checkstyle.RequireBundleCheck">
       <property name="severity" value="error" />
    </module>
+   
+   <module name="org.openhab.tools.analysis.checkstyle.AboutHtmlCheck">
+     <property name="severity" value="error" />
+     <!-- You can give a path to a valid about.html file with the property "validAboutHtmlFilePath", e.g.
+          <property name="validAboutHtmlFilePath" value="..." />
+          The processed about.html files will be compared with this file.
+          If you don't provide such property or the path is incorrect,
```

Comment:
There might be a better option. We might load the content for a valid about.html file from a URL, not from a local path to the file. 

@kaikreuzer, are you OK to load the content form some of the already existing about.html files in openhab2-addons repository? If you accept this, the configuration property will look something like this:
`<property name="validAboutHtmlFileURL" value="https://raw.githubusercontent.com/openhab/openhab2-addons/master/addons/ui/org.openhab.ui.cometvisu.php/about.html" />`